LayoutTests/svg/dom/SVGGeometryElement-getTotalLength-detached.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<title>SVGGeometryElement.getTotalLength method (element detached)</title>
+<script src="../../resources/testharness.js"></script>
+<script src="../../resources/testharnessreport.js"></script>
+<script>
+test(function() {
+  var pathElement = document.createElementNS("http://www.w3.org/2000/svg", "path");
+
+  function getTotalLength(string) {
+      pathElement.setAttribute("d", string);
+      return pathElement.getTotalLength();
+  }
+
+  assert_equals(getTotalLength('M0,20 L400,20 L640,20'), 640);
+  assert_equals(getTotalLength('M0,20 L400,20 L640,20 z'), 1280);
+  assert_equals(getTotalLength('M0,20 L400,20 z M 320,20 L640,20'), 1120);
```

Comment:
Can we add a test for the empty path case:
```
assert_throws_dom("InvalidStateError", function() { getTotalLength('') });
```

Reply:
Added.